app/models/bookmarked_post.rb

```diff
@@ -0,0 +1,6 @@
+class BookmarkedPost < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+
+  validates :user_id, :post_id, presence: true
```

Comment:
Man ir lielas aizdomas kā šis validacijas jau strāda pēc defaulta, tāpēc kā tev ir reference.
Pamegini no Rails konsoles bez šis validacijas izveidot ierakstu, saņemsi tadu pati erroru